js/display-pictures.ts

```diff
@@ -0,0 +1,25 @@
+/* eslint-disable @typescript-eslint/no-non-null-assertion */
+import { Description } from './mock.ts';
+
+const pictureList = document.querySelector('.pictures');
```

Comment:
`querySelector` по сигнатуре ппринимает дженерики, в которых можно уточнять ожидаемый тег
![image](https://github.com/htmlacademy-javascript/2154345-kekstagram-29/assets/47776594/f633d73b-8eb5-4e2b-aea9-206add10e0a1)

Поскольку отдельного стандартного типа на секции нет, можем сказать, что это див (или ничего в дженерик не передовать)
```suggestion
const pictureList = document.querySelector<HTMLDivElement>('.pictures');
```